Detectors/AOD/src/AODProducerWorkflowSpec.cxx

```diff
@@ -499,8 +535,8 @@ void AODProducerWorkflowDPL::addToFwdTracksTable(FwdTracksCursorType& fwdTracksC
     chi2matchmchmid = track.getMIDMatchingChi2();
     chi2matchmchmft = track.getMFTMCHMatchingChi2();
     matchscoremchmft = track.getMFTMCHMatchingScore();
-    matchmfttrackid = track.getMFTTrackID();
-    matchmchtrackid = track.getMCHTrackID();
+    matchmfttrackid = mIndexTableMFT[track.getMFTTrackID()];
```

Comment:
here you fill the remapped index of the MFT track contributing to the match, but you actually not registering it in the fillIndexTablesPerCollision.

Reply:
Same, not sure I understand the comment. All MFT tracks are registered in fillIndexTablesPerCollision, including the ones contributing to the match. Maybe I am missing something.

Reply:
In the fillIndexTablesPerCollision you fill the mGIDToTableMFTID only for standalone MFT tracks: https://github.com/AliceO2Group/AliceO2/blob/b2699abec433f6ed70585650719058455cd61e1f/Detectors/AOD/src/AODProducerWorkflowSpec.cxx#L362, see the comment below.

Reply:
From what I tested this effectively seems to fill all MFT tracks. The structure inside this `fillIndexTablesPerCollision` is the same as in the `fillTrackTablesPerCollision`function for MFT tracks.